apps/supercode-cli/server/.env.example

```
@@ -85,7 +85,7 @@ FIRECRAWL_API_KEY=""
 DODO_PAYMENTS_API_KEY=""
 DODO_PAYMENTS_WEBHOOK_KEY=""
 # ⚪ "test" to use test-mode product IDs in the seed; omit for live.
-DODO_MODE="test"
+DODO_MODE=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)